server/dataParser/cart.js

```diff
@@ -0,0 +1,27 @@
+require('dotenv').config();
+
+const axios = require('axios');
+const Promise = require('bluebird');
+let url = 'https://app-hrsei-api.herokuapp.com/api/fec2/rfp';
+
+var config = {
```

Comment:
The URL and config are being duplicated in a bunch of files, so one simple solution to prevent you from having to set this up in each one of these files is to use `axios.create()`. (See: https://axios-http.com/docs/instance) With this, you can create an instance object that remembers your base URL and your headers/config. You can then pass around this instance object and call `.get()`, `.push()`, etc. on it without worrying about the base URL or authorization token.